test/alternator/util.py

```diff
@@ -206,3 +209,22 @@ def client_no_transform(client):
 
 def is_aws(dynamodb):
     return dynamodb.meta.client._endpoint.host.endswith('.amazonaws.com')
+
+# Tries to inject an error via Scylla REST API. It only works on Scylla,
+# and only in specific build modes (dev, debug, sanitize), so this function
+# will trigger a test to be skipped if it cannot be executed.
+@contextmanager
+def scylla_inject_error(dynamodb, err, one_shot=False):
+    if dynamodb.meta.client._endpoint.host.endswith('.amazonaws.com'):
+        pytest.skip("Error injection not enabled on AWS")
+    url = dynamodb.meta.client._endpoint.host
+    # The REST API is on port 10000, and always http, not https.
+    url = re.sub(r':[0-9]+(/|$)', ':10000', url)
+    url = re.sub(r'^https:', 'http:', url)
+    response = requests.post(f'{url}/v2/error_injection/injection/{err}?one_shot={one_shot}')
+    response = requests.get(f'{url}/v2/error_injection/injection')
```

Comment:
nitpick (not important enough to delay this series): can we read just {err} and not all the injections?

Reply:
no, we only have POST and DELETE ops for a single injection point. It would be nice indeed to read a single point and all details about it, but it's not here yet